src/npc.cpp

```diff
@@ -1172,7 +1172,7 @@ void npc::make_angry()
         return; // We're already angry!
     }
 
-    add_msg( m_debug, "%s gets angry", name.c_str() );
+    add_msg( "%s gets angry", name.c_str() );
```

Comment:
If it is to be kept, it should also get a sight check, to prevent offscreen NPCs announcing their presence with it.